source/agora/test/Ledger.d

```diff
@@ -63,6 +64,7 @@ unittest
 
         block_txes ~= txs.sort.array;
         last_txs = txs;
+        Thread.sleep(500.msecs);
```

Comment:
Please add a comment as to why it's required

Reply:
Might not be necessary anymore, I'll try to remove it.